qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -280,3 +304,46 @@ def test_get_to_console(self):
 #            log.info('ls -a succeeded')
 #        else:
 #            log.info('ls -a failed')
+
+    class TestCD(CephFSTestCase):
+        CLIENTS_REQUIRED = 1
+
+        def setUp(self):
+            global mount_x
+
+            CephFSTestCase.setUp(self)
+            mount_x = self.mount_a
+
+        def tearDown(self):
+            global mount_x
+            mount_x = None
+
+        def test_cd_with_no_args(self):
+            """
+            Test that when cd is issued without any arguments, CWD is changed
+            to root directory.
+            """
+            path = 'dir1/dir2/dir3'
+            self.mount_a.run_shell('mkdir -p ' + path)
+            expected_cwd = '/'
+
+            script = 'cd {}\ncd\ncwd\n'.format(path)
+            output = run_cephfs_shell_script(script).stdout.getvalue().strip()
+            assertion_errmsg = 'actual output - ' + output + '\n' + \
+                               'expected output - ' + output + '\n'
+            assert output == expected_cwd, assertion_errmsg
```

Comment:
Instead of this, use self.assertEqual(...).

Reply:
Ok. I've made the change.